### DISCUSSION TITLE
Sign in problem

### DISCUSSION THREAD

**Discussion by maximN27** [z-huang/music#1663] [Q&A] [answered]
Guys why is it asking me to sign in? ![Screenshot_20241024_064935.jpg](https://github.com/user-attachments/assets/74440c51-34ec-4cda-9351-430fdebc8f1a)

**Comment by Jolt32** [comment, marked as answer]
@TheMaximusPrime found it just login here, will say "Log in" on that second screenshot for you.
![Screenshot_20241024-134602_InnerTune](https://github.com/user-attachments/assets/699d4cb9-6289-426c-8d1a-61e492410a46)
![Screenshot_20241024-134635_InnerTune](https://github.com/user-attachments/assets/56621221-5ee9-491b-9c52-14ff8fb99aef)

**Reply by beswalod** [reply]
Same

**Reply by Jolt32** [reply]
> Same

Just need to sign in to your Google account in the app, look at screenshots above.
